### ISSUE TITLE
predicting zero objects always

### ISSUE THREAD

**Issue by ask142857** [YunYang1994/tensorflow-yolov3#214]
I am clearly observing decrease in train loss and test loss while training but when I'm running evaluate.py it  just prints ground truth , i have put some print statements and realized that for each image the predict function in evaluate is giving list of zero size implying it has detected zero objects. Help me fix this issue

**Comment by YunYang1994** [comment]
hello, anybody here ?
Can anybody fix this issue on the basis of such descriptions ?

**Comment by ask142857** [comment]
I have run the model on a random rbg image to check whether it's giving any output from the evaluate.py . It has given the same empty array as output from the predict function . Is everything alright with evaluate.py?

**Comment by lway-hb** [comment]
me too

**Comment by dvlee1024** [comment]
Me too. I am using my own dataset which has 2 classes.  Still predice zero objects.

 I did not modify any value in config.py except `__C.TRAIN.ANNOT_PATH` and `__C.YOLO.CLASSES`. I used yolov3_coco_demo as my initial weight and the loss on the beginning was 2716. Is the loss normal?

**Comment by kelvinliu04** [comment]
I got this problem, then change self.trainable: True in evaluate.py. In my case, it's solved
